code/datums/wounds/_wounds.dm

```diff
@@ -445,7 +494,21 @@
 		if(WOUND_SEVERITY_CRITICAL)
 			return "Critical"
 
+
+/// Returns false if our limb is the head or chest, true otherwise.
+/datum/wound/proc/limb_unimportant()
```

Comment:
Important in what sense? Can you expand on this?

Reply:
just renamed it to make more sense